src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_validators.py

```diff
@@ -0,0 +1,21 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+import argparse
+
+# COMMAND NAMESPACE VALIDATORS
+
+def process_policy_namespace(ns):
+    # TODO Consider supporting mutual exclusion in CLI
+    # https://docs.python.org/2/library/argparse.html#mutual-exclusion
+    if sum(1 for p in [ns.object_id, ns.spn, ns.upn] if p) != 1:
+        raise argparse.ArgumentError(
+            None, 'One of the arguments --object-id --spn --upn is required')
```

Comment:
>One of the arguments --object-id --spn --upn is required [](start = 19, length = 56)

This error message is not quite clear for the case where more than one options are specified. I would add a second error message for if sum > 2 that says "Please specify only -a, -b, or -c."